### PR TITLE
Add SEND_EMAILS environment variable

### LINKED ISSUE #109: Add setting for disabling emails in development
Add an environment variable that will disable the sending of emails in development.

If you for example export the production database and use it locally, you wouldn't want the system to send out emails to users about stuff that hasn't happened on the production server...

### PR DESCRIPTION
**Pull request by elmaxe**
Also created a single send email function.

Fixes: #109

Continuation of #113

### CODE REVIEW

**Comment by Herkarl** [comment]
Seems to include commits from #116 , is that intentional?

**Comment by elmaxe** [comment]
> Seems to include commits from #116 , is that intentional?

No, trying to fix

**Comment by elmaxe** [comment]
Fixed @Herkarl

**Review by Herkarl — APPROVED** [review]
Lgtm

File: .env.example [hunk 1]
```diff
@@ -7,4 +7,5 @@ S3_ACCESS_KEY_ID=super_secret
 S3_SECRET_ACCESS_KEY=super_secret
 S3_USE_SIGV4=True
 S3_HOST=s3.eu-west-1.amazonaws.com
-SPAM_API_KEY=super_secret
\ No newline at end of file
+SPAM_API_KEY=super_secret
+SEND_EMAILS=True
```
Comment (niklasvatn) [inline .env.example:11]:
```suggestion
SEND_EMAILS=True

```

**Pull request merged**